source/routing/loading-and-error-substates.md

```diff
@@ -1,6 +1,9 @@
 The Ember Router allows you to provide feedback that a route is loading, as well
 as when an error occurs in loading a route.
 
+The routes for `error` and `loading` are implicitly defined for you, so they
```

Comment:
Can you elaborate a bit more? The crux is that they are sub-states of a route, not exactly routes themselves, that is why they won't show in the router map.

Reply:
updated!